Add data durability documentation to Dedicated and Serverless (#5105)
influxdata/docs-v2: content/influxdb/cloud-dedicated/reference/internals/durability.md

reference the newly created Parquet files. If an ingester is gracefully shut
down (for example, during a new software deployment), it flushes the contents of
the WAL to the Parquet files before shutting down.

## Backups

{{< cloud-name >}} implements to following data backup strategies:

- **Backup of WAL file**: The WAL file is written on locally-attached storage.
  If an ingester process fails, the new ingester simply reads the WAL file on
  startup and continues normal operation. WAL files are maintained until their
  contents have been written to the Parquet files in object storage.
  For added protection, ingesters can be configured for write replication, where
  each measurement is written to two different WAL files before acknowledging
  the write.

- **Backup of Parquet files**: Parquet files are stored in object storage where
  they are redundantly stored on multiple devices across a minimum of three
  availability zones in a cloud Region. Parquet files associated with each
  database are kept in object storage for the duration of database retention period
  plus an additional time period (approximately 100 days).

> - **Backup of catalog**: InfluxData keeps a transaction log of all recent updates
>   to the [InfluxDB catalog](#influxdb-catalog). We also generate a daily backup of
>   the catalog and preserve them for at least 100 days in object storage across a minimum
>   of three availability zones.

Reviewer: ```suggestion
- **Backup of catalog**: InfluxData keeps a transaction log of all recent updates
  to the [InfluxDB catalog](#influxdb-catalog) and generates a daily backup of
  the catalog. Backups are preserved for at least 100 days in object storage across a minimum
  of three availability zones.
```
